src/UnitTests/XpressionMapperTests.cs

```diff
@@ -212,6 +212,27 @@ public void Map_object_when_null_values_are_typed()
         }
 
         [Fact]
+        public void Map_expression_should_throws_exception_when_mapping_is_missed()
+        {
+            // Arrange
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<Car, CarModel>();
+            });
+
+            var customMapper = config.CreateMapper();
+
+            Expression<Func<Car, bool>> expression = car => car.Year == 2017;
+
+            // Act
+            var exception = Assert.Throws<AutoMapperMappingException>(() => customMapper.MapExpression<Expression<Func<CarModel, bool>>>(expression));
+
```

Comment:
If I call Map instead of MapExpression I get another exception:

typeof(AutoMapper.AutoMapperMappingException): Error mapping types.

Mapping types:
Expression`1 -> Expression`1
System.Linq.Expressions.Expression`1[[System.Func`2[[AutoMapper.UnitTests.Car, AutoMapper.UnitTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null],[System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]] -> System.Linq.Expressions.Expression`1[[System.Func`2[[AutoMapper.UnitTests.CarModel, AutoMapper.UnitTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null],[System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089